dipy/viz/actor.py

```diff
@@ -0,0 +1,661 @@
+# from __future__ import division, print_function, absolute_import
+
+import numpy as np
+
+from dipy.viz.colormap import line_colors
+from dipy.viz.utils import numpy_to_vtk_points, numpy_to_vtk_colors
+from dipy.viz.utils import set_input, map_coordinates_3d_4d
+from dipy.core.ndindex import ndindex
+
+# Conditional import machinery for vtk
+from dipy.utils.optpkg import optional_package
+
+# Allow import, but disable doctests if we don't have vtk
+vtk, have_vtk, setup_module = optional_package('vtk')
+colors, have_vtk_colors, _ = optional_package('vtk.util.colors')
+numpy_support, have_ns, _ = optional_package('vtk.util.numpy_support')
+
+if have_vtk:
+
+    version = vtk.vtkVersion.GetVTKSourceVersion().split(' ')[-1]
+    major_version = vtk.vtkVersion.GetVTKMajorVersion()
+
+
+def colormap_lookup_table(scale_range=(0, 1), hue_range=(0.8, 0),
```

Comment:
This is not an actor, we should move it in another file, maybe `viz/colormap.py` ?